omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -1191,16 +1188,33 @@
                 var appointment = new Date((jq("#returndate").find('input[type=hidden]').val()));
                 var transferOutDate = new Date((getValue('transfer_out_date.value')));
 
+                if(getValue('returndate.value')==""){
+                    jq().toastmessage('showErrorToast', "The return date is required");
```

Comment:
@semujju The message is "The return visit date is required" - however why do you not just put a required attribute on the field

Reply:
When I used required="required", it wasn`t returning a message below the box that is why I used js. 
However, I have tested with required=true and it solves the problem. So I will remove the js